repositories.md

```diff
@@ -39,37 +39,37 @@ Listing the public key of the Time Server in Director Targets metadata is necess
 
 1. Gather the tokens from each secondary ECU's version report.
-2. Send the list of tokens to the Time Server to fetch the current time. The Time Server responds as described in the [Time Server section](#time-server), providing a cryptographic attestation of the last known time.
-3. If the Time Server's response meets the criteria below, update the Primary ECU's clock and retain the Time Server's response for distribution to Secondary ECUs. Otherwise, the response will be discarded and procedure continues without an updated time.  The criteria for checking the Time Server's response are:
+2. Send the list of tokens to the Time Server to fetch the current time. The Time Server responds, as described in the [Time Server section](#time-server), by providing a cryptographic attestation of the last known time.
+3. If the Time Server's response meets the criteria below, update the Primary ECU's clock and retain the Time Server's response for distribution to Secondary ECUs. If it fails to meet this criteria, discard the response and continue the procedure without an updated time.  The criteria for checking the Time Server's response are:
   - The signature over the Time Server's response is valid.
-  - All the tokens provided to the Time Server have been included in the response.
+  - All the tokens provided to the Time Server are included in the response.
   - The time in the Time Server's response is later than the last time verified in this manner.
 
 #### ECU Version Report
 
-The ECU version report from each Secondary will contain a token to be sent to the Time Server. This token SHOULD be unique for each update cycle to prevent a replay. Since we expect updates to be relatively infrequent (e.g., due to limited number of write cycles), and that there will be a large number of possible tokens, it should be possible to produce a unique token for every update.
+The ECU version report from each Secondary will contain a token to be sent to the Time Server. To prevent a replay, this token SHOULD be unique for each update cycle. This stipulation should be possible to achieve as we expect updates to be relatively infrequent (e.g., due to limited number of write cycles), and that there will be a large number of possible tokens.
 
-The payload of the ECU version report sent to the Director might contain the token sent to the Time Server. This token is in the version report sent from Secondaries to the Primary, and so is in the signed version of the version report. If the token is removed, the signature will not match.
+The payload of the ECU version report sent to the Director may contain the token sent to the Time Server. This token is part of the version report sent from Secondaries to the Primary. If the token is removed, the signature will not match.
```

Comment:
I found these few sentences rather snarly. I changed the might to me, because to me its use suggested also that it might not contain this token. If the token is needed to match the signature, then why is this not a should? What actually is the key point that needs to be made here? Any suggestions on how we can change this?

Reply:
Here's my understanding: the previous paragraph describes how the tokens from Secondaries get to the Primary. This bit describes how the Primary sends those tokens and its own to the Time Server. One option is to include them as part of the ECU version report sent to the Director. They could be sent in some other way, but it seems simplest to do it that way. Hence, I think "MAY" is warranted, but I think we should be talking about "tokens", not just one "token". The second sentence ("This token is part of the version report sent from Secondaries to the Primary.") seems redundant with the preceding paragraph. And then the third sentence can be revised to make it clear that an advantage of putting the token(s) in the ECU version report is that the signature then includes that and protects against removing it. That paragraph can probably be merged with the preceding one. I'd like to suggest a change to do all that, but since it covers two paragraphs, I can't, and I don't have rights to push to your branch. I can open a PR against your own PR if that would help.

Reply:
How about I commit this PR and then you clarify this point in a separate one? It seems simpler that way.

Reply:
Yeah, that's fine.

Reply:
See https://github.com/uptane/deployment-considerations/pull/32.